puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/tfmROR51CoprCleanup.groovy

```diff
@@ -0,0 +1,40 @@
+pipeline {
+    agent { label 'rpmbuild' }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        ansiColor('xterm')
+        buildDiscarder(logRotator(numToKeepStr: '15'))
+    }
+
+    triggers {
+        cron('0 0 * * *')
+    }
+
+    stages {
+        stage('Clone Packaging') {
+            steps {
+
+                deleteDir()
+                git url: 'https://github.com/theforeman/tfm-ror51-packaging', branch: 'master'
```

Comment:
In many cases I wondered if we can reuse the github -> url property here to reduce duplication.